keyboards/keebwerk/nano_slider/keymaps/default/keymap.c

```diff
@@ -74,14 +74,9 @@ bool process_record_user(uint16_t keycode, keyrecord_t *record) {
 
 uint8_t divisor = 0;
 
-void slider(void) {
-    if (divisor++) { // only run the slider function 1/256 times it's called
-        return;
-    }
-
-    midi_send_cc(&midi_device, 2, 0x3E, 0x7F - (analogReadPin(SLIDER_PIN) >> 3));
-}
-
 void matrix_scan_user(void) {
-    slider();
+  if (divisor++) { // only run the slider function 1/256 times it's called
```

Comment:
Please revert these changes.

Reply:
reverted in subsequent commit.

Reply:
I see no reason why this PR should touch this file.